### ISSUE TITLE
Consider upgrading to Angular 1.4

### ISSUE THREAD

**Issue by nfantone** [cgross/angular-notify#38]
Is there any reason why Angular `~1.4` wouldn't work? Right now, this is the only dependency on my current project that forces me to use `"resolutions"` on `bower.json`.

**Comment by pruge** [comment]
in my case, no problem
angular 1.4.3
angular-notify 2.5.0

**Comment by nfantone** [comment]
Yes, no problem here either. It's just that the dependency is capped at 1.3 and it forces users to add a resolution to avoid conflicting versions of angular. This is particularly bothersome in CI and deploy scripts.

**Comment by GustavSt** [comment]
With $httpProvider useLegacyPromiseExtension set to false i get errors because notify uses .success-syntax with $http. Maybe something to refactor as well

**Comment by mailinger-mate** [comment]
I am also looking forward to the update of dependency version.

**Comment by Poltuu** [comment]
I am looking forward to the update of dependency version as well.
